frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -244,6 +243,14 @@ export const ImageManifestVulnTableRow: RowFunction<ImageManifestVuln> = ({
         {Object.keys(obj.status.affectedPods).length}
       </TableData>
       <TableData className={tableColumnClasses[4]}>{obj.status.fixableCount || 0}</TableData>
+      <TableData className={tableColumnClasses[4]}>
+        {_.sum([
+          obj.status.highCount,
+          obj.status.mediumCount,
+          obj.status.lowCount,
+          obj.status.unknownCount,
```

Comment:
can destructure it(`highCount` , `mediumCount` etc) above and good to have check for status and check unknownCount and other count is defined as we are doing sum.

and maybe we can avoid lodash